migration.php

```diff
@@ -0,0 +1,160 @@
+    $model_map = array(
+        'fanvil-X3S' => 'fanvil-X3SG',
+        'fanvil-X4' => 'fanvil-X4',
+        'fanvil-X5S' => 'fanvil-X5',
+        'fanvil-X6' => 'fanvil-X6',
+        'gigaset-Maxwell-2' => 'gigaset-Maxwell2',
+        'gigaset-Maxwell-3' => 'gigaset-Maxwell3',
+        'gigaset-Maxwell-Basic' => 'gigaset-MaxwellBasic',
+        'sangoma-S205' => 'sangoma-S205',
+        'sangoma-S206' => 'sangoma-S206',
+        'sangoma-S300' => 'sangoma-S300',
+        'sangoma-S305' => 'sangoma-S305',
+        'sangoma-S400' => 'sangoma-S400',
+        'sangoma-S405' => 'sangoma-S405',
+        'sangoma-S406' => 'sangoma-S406',
+        'sangoma-S500' => 'sangoma-S500',
+        'sangoma-S505' => 'sangoma-S505',
+        'sangoma-S700' => 'sangoma-S700',
+        'sangoma-S705' => 'sangoma-S705',
+        'snom-D710' => 'snom-D710',
+        'snom-D715' => 'snom-D715',
+        'snom-D725' => 'snom-D725',
+        'yealink-T19' => 'yealink-T19',
+        'yealink-T19P_E2' => 'yealink-T19',
+        'yealink-T21P' => 'yealink-T21',
+        'yealink-T21P_E' => 'yealink-T21',
+        'yealink-T23P_G' => 'yealink-T23',
+        'yealink-T27G' => 'yealink-T27',
+        'yealink-T27P' => 'yealink-T27',
+        'yealink-T29G' => 'yealink-T29',
+        'yealink-T40G' => 'yealink-T40',
+        'yealink-T40P' => 'yealink-T40',
+        'yealink-T41P' => 'yealink-T41',
+        'yealink-T41S' => 'yealink-T41',
+        'yealink-T42G' => 'yealink-T42',
+        'yealink-T42S' => 'yealink-T42',
+        'yealink-T46G' => 'yealink-T46',
+        'yealink-T46S' => 'yealink-T46',
+        'yealink-T48G' => 'yealink-T48',
+        'yealink-T48S' => 'yealink-T48',
+        'yealink-T49G' => 'yealink-T49',
+        'yealink-T52S' => 'yealink-T52',
+        'yealink-T53' => 'yealink-T53',
+        'yealink-T53W' => 'yealink-T53',
+        'yealink-T54S' => 'yealink-T54',
+        'yealink-T54W' => 'yealink-T54',
+        'yealink-T56A' => 'yealink-T56',
+        'yealink-T57' => 'yealink-T57',
+        'yealink-T57W' => 'yealink-T57',
+        'yealink-T58A' => 'yealink-T58',
+        'yealink-VP59' => 'yealink-VP59'
+    );
+
+    if (isset($model_map['oldmodel'])){
+        $model = $model_map['oldmodel'];
+    }
+
+    # Add phones to tancredi
+    $scope = new \Tancredi\Entity\Scope($mac, $storage, $logger);
+    $scope->metadata['displayName'] = $brand;
+    $scope->metadata['inheritFrom'] = $model;
+    $scope->metadata['scopeType'] = 'phone';
+    $scope->setVariables($variables);
+    \Tancredi\Entity\TokenManager::createToken(uniqid($prefix = rand(), $more_entropy = TRUE), $mac , TRUE); // create first time access token
+    \Tancredi\Entity\TokenManager::createToken(uniqid($prefix = rand(), $more_entropy = TRUE), $mac , FALSE); // create token
+    $phone_scope = \Tancredi\Entity\Scope::getPhoneScope($mac, $storage, $logger);
+
+    # Configure RPS with Falconieri
+    setFalconieriRPS($mac, $phone_scope['provisioning_url1']);
```

Comment:
I'd like to have the same behavior of UI. If Falconieri fails it shows a warning.

Can we check the Falconieri response status and log a warning if it fails?

Reply:
Falconieri error_log output is printed to standard output. We could also print it to Tancredi log.

Reply:
I'd expect the log is sent (also) to stdout